deployment/docker-compose/docker-compose.sh

```diff
@@ -139,6 +139,7 @@ echo -e "
 Name: api-server
 Host: 0.0.0.0
 Port: 8888
+MaxPendingTxCount: 10000
```

Comment:
Can we have a module like `TxPool`, and put `MaxPendingTxCount` under it, I kind dislike `MaxPendingTxCount` to be parallel with `Host` and `Port`